package.json

```diff
@@ -33,7 +33,7 @@
   },
   "peerDependencies": {
     "n-ui-foundations": "^9.0.0",
-    "@financial-times/n-myft-ui": "^28.3.0"
+    "@financial-times/n-myft-ui": "^28.3.0 || 29.x"
```

Comment:
@kavanagh if we do that and a breaking change for n-topic-flag is introduced in v30+, wouldn't a consuming app be able to upgrade n-myft-ui and unwittingly break their own implementation of n-topic-flag?

Reply:
Yes, possibly. I mention it because I know of a couple of bits of work in development that may result in a major version bump to n-myft-ui (potentially in a matter of days). I'm happy to go with your change if you want, but the risk is you may have to repeat this.